frontend/src/index.css

```
@@ -21,15 +23,42 @@
   --text-on-primary: #FFFFFF;
   /* White text on blue buttons */
 
+  /*stat colors */
```

Stylelint (17.2.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)